.circleci/config.yml

```diff
@@ -11,7 +12,17 @@ jobs:
       - node/install-packages
       - run: npm install
       - run: npm test
+  python-build-and-test:
+    executor: python/default
+    steps:
+      - checkout
+      - python/install-deps
+      - run: pip install -r requirements.txt
```

Comment:
This is taken care of by the line above

```suggestion
```

Reply:
Done!
:D I just wanted to be sure I've covered everything that was in scope for this issue